### PR TITLE
fricas 1.3.10 (new formula)

### PR DESCRIPTION
**Pull request by fahasch**
FriCAS is a free advanced computer algebra system that is actively developed. This is a new formula that allows compilation of the latest version as well as of HEAD. A test testing the CAS capabilities is included.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/f/fricas.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      ENV.deparallelize
+      system "../configure", *std_configure_args
+      system "make"
+      system "make", "install"
+    end
+  end
+
+  test do
+    assert_match %r{ \(/ \(pi\) 2\)\n},
+      shell_output("echo 'integrate(sqrt(1-x^2),x=-1..1)::InputForm' | #{bin}/fricas -nosman")
```
Comment (SMillerDev) [inline Formula/f/fricas.rb:30]:
```suggestion
    assert_match %r{ \(/ \(pi\) 2\)\n},
      pipe_output('integrate(sqrt(1-x^2),x=-1..1)::InputForm', bin/"fricas -nosman")
```

File: Formula/f/fricas.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  depends_on "sbcl"
+
+  def install
+    Dir.mkdir("build")
+    chdir "build" do
+      ENV.deparallelize
+      system "../configure", *std_configure_args
+      system "make"
+      system "make", "install"
+    end
```
[the remark this answers is not in the document]
Comment (fahasch) [reply Formula/f/fricas.rb:25]: On MacOS (I suppose due to the fact that the filesystem is case insensitive), it is not possible to build it in the tree.

File: Formula/f/fricas.rb (hunk 1 above)
Comment (fahasch) [reply Formula/f/fricas.rb]: I have updated the formula following your suggestion.

**Comment by fahasch** [comment]
There is the strange error in the linux build (when executing the test):
> fatal error encountered in SBCL pid 29340 tid 29340:
  Can't find sbcl.core
  
This does not happen on my local build. If feels like `sbcl` is not properly set up. Any thoughts?

**Pull request closed without merging**